### DISCUSSION TITLE
Login problem in different devices

### DISCUSSION THREAD

**Discussion by 1235shahid** [community/community#144195] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I have an account in GitHub which I opened using my phone but when I try to login using the browser of my team members phone . Then it sends verification code in  my Gmail ( which I used to open the account ) which is in my phone. But I don't had my phone then.

How to login in my friends mobiles browser easily without needing my phone.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @1235shahid ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
